package/MDAnalysis/core/universe.py

```diff
@@ -449,6 +462,12 @@ def impropers(self):
         """Improper dihedral angles between atoms"""
         return self.atoms.impropers
 
+    def __hash__(self):
```

Comment:
I defined a `Universe.__hash__` here to use for the anchor storage retrieval, but that does open a whole can of worms about what hashing a Universe actually means.  It might be safer to revert this and instead call this `_generate_anchor_hash` rather than claim it works as a proper hash for all purposes.